src/plugins/osmosis/index.tsx

```diff
@@ -0,0 +1,36 @@
+import { ChainAdapter as OsmosisChainAdapter } from '@shapeshiftoss/chain-adapters/dist/cosmossdk/osmosis/OsmosisChainAdapter'
+import { ChainTypes } from '@shapeshiftoss/types'
+import * as unchained from '@shapeshiftoss/unchained-client'
+import { getConfig } from 'config'
+import { Plugins } from 'plugins'
+
+export function register(): Plugins {
+  return [
+    [
+      'osmosisChainAdapter',
+      {
```

Comment:
needs a feature flag, or osmosis will appear in the portfolio when this is merged

https://github.com/shapeshift/web/blob/develop/src/plugins/index.ts#L15